charts/iom/.bumpversion.cfg

```diff
@@ -0,0 +1,9 @@
+[bumpversion]
+current_version = 2.1.0
+commit = true
+tag = false
+message = Bump IOM chart version: {current_version} → {new_version}
+
+#[bumpversion:file:Chart.yaml]
```

Comment:
This will deactivate automatic version number replacement in `Chart.yaml` . If that is what you want, ok. But it will still bump the `current_version` field in this file.